introduction.md

```diff
@@ -1,3 +1,6 @@
 Jonathon Schloss
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?